packages/serverpod_test/lib/src/with_serverpod.dart

```diff
@@ -66,6 +66,7 @@ void Function(TestClosure<T>)
   required RollbackDatabase? maybeRollbackDatabase,
   required bool? maybeEnableSessionLogging,
   required List<String>? maybeTestGroupTagsOverride,
+  required Duration? maybeServerpodStartTimeout,
```

Comment:
@SandPod I actually think that should be in the generated code to be helpful via intellisense. Otherwise it's just for curious people that follow the code into the library. I can create an issue for that as well.

Reply:
Yes, makes sense! I might have commented on the wrong place. But I think it makes sense to provide documentation for the interface as well :)